src/components/About/Clients.module.css

```diff
@@ -10,6 +10,7 @@
   align-items: center;
   height: 632px;
   position: relative;
+  overflow-y: hidden;
```

Comment:
I added this to prevent the image from bleeding into the Director's component at mobile size.
<img width="740" alt="Screenshot 2024-07-10 at 11 49 11 AM" src="https://github.com/poulami-saha/Global-Code-Queens-MyTeam-Website/assets/6990439/9192caa0-59a9-4cfb-8269-3c1b5f69e52c">